### PR TITLE
Migrate from ESRI

### PR DESCRIPTION
**Pull request by PeterPetrik**

### CODE REVIEW

**Comment by alex-cit** [comment]
Some changes in the text, added links to relevant pages in the docs / to QGIS page, changed section names a bit.
- ESRI is officially Esri
- added a tip at the beginning of the page about getting familiar with Mergin Maps. All the info is on the bottom of the page, but I feel like it should be also on the top (some people may not scroll so far down to read it :) )
- How I read the https://north-road.com/slyr/ , the fcommunity version offers only converting *.style and *.lyr files (not project files). Please check, maybe I misunderstood.
- added links/steps to downloading Mergin Maps components to the Migrate section, as I feel like it might be unclear that they need to use the plugin and have an account to use the app
- ...

File: src/migrate/arcgis/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+To use your QGIS project with the <MainPlatformNameLink /> platform:
+1. [Sign up to <MainPlatformName />](../../setup/sign-up-to-mergin-maps/)
+2. [Install the <QGISPluginName />](../../setup/install-mergin-maps-plugin-for-qgis/)
+3. [Install the <MobileAppName />](../../setup/install-mobile-app/)
+4. Sync the QGIS project to <MobileAppName /> using the <QGISPluginNameShort />. See how the settings done in QGIS translate to the <MobileAppNameShort />.
+
+### SLYR 
+
+The [SLYR tool](https://north-road.com/slyr/) by [North Road](https://north-road.com) facilitates the migration from ArcGIS to QGIS by enabling the conversion of various proprietary data formats, styles and project files to QGIS-compatible formats, such as the conversion of MXD (ArcMap) and AXL (<NoSpellcheck id="ArcIMS" />) files to QGIS project files.
```
Comment (nyalldawson) [inline src/migrate/arcgis/index.md:51]:
```suggestion
The [SLYR tool](https://north-road.com/slyr/) by [North Road](https://north-road.com) facilitates the migration from ArcGIS to QGIS by automatically converting various proprietary ESRI data formats, styles and project files to QGIS-compatible formats, such as the conversion of MXD (ArcMap) and APRX (ArcGIS Pro) files to QGIS project files.
```

File: src/migrate/arcgis/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+This tool helps users transition smoothly by preserving complex symbology, layouts, and project structure. It simplifies the shift from Esri tools to the open-source QGIS environment, reducing the time and effort needed to replicate existing workflows. 
+
+SLYR tool offers Community and Licensed version:
+- Community version is open-sourced on <GitHubRepo id="north-road/slyr" desc="GitHub" /> and can be freely installed via QGIS Plugin Manager to extract, parse, and convert Esri `.lyr` and `.style` files to QGIS-compatible formats. 
+- Licensed version is closed-sourced and has features such as converting `MXD/MXT` project files to `QGS` or converting various data formats to GeoPackage.
```
Comment (nyalldawson) [inline src/migrate/arcgis/index.md:57]:
```suggestion
- Licensed version is closed-sourced and has features such as converting `MXD/MXT` project files to `QGS` or converting various data formats to GeoPackage, and support for exporting QGIS projects to APRX for use within ArcGIS Pro.
```

File: src/migrate/arcgis/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- Licensed version is closed-sourced and has features such as converting `MXD/MXT` project files to `QGS` or converting various data formats to GeoPackage. 
+
+Full comparison of the version can be found on North Road's [SLYR project page](https://north-road.com/slyr/). 
+
+![SLYR logo](./slyr.png "SLYR logo")
+
+
+### Domains 
+
+ArcGIS Pro [domains](https://pro.arcgis.com/en/pro-app/latest/help/data/geodatabases/overview/create-modify-and-delete-domains.htm) are used to restrict the valid values allowed in an attribute field. Convert them manually to QGIS drop-down widgets - likely [Value Relation](../../layer/form-widgets) - could be time consuming. You can export the domains to CSV to be imported and set value relation widgets, along with it's custom expression visibility to mimic cascading forms. We again recommend using [SLYR](#slyr) to facilitate the conversion of domains to QGIS.
```
Comment (nyalldawson) [inline src/migrate/arcgis/index.md:66]: This should actually all happen automatically in recent qgis /gdal versions, without the need for slyr 🥳. It's all available out of the box.

File: src/migrate/arcgis/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+### Data Formats
+
+ArcGIS Pro [supports GeoPackages](https://pro.arcgis.com/en/pro-app/latest/help/data/databases/work-with-sqlite-databases-in-arcgis-pro.htm), which are the primary and recommended format for your survey layer in <MainPlatformName />. Both ArcGIS and QGIS can also handle Shapefile, GeoTIFF, GeoJSON, WMS, WFS, and PostGIS layers. However, we do not recommend using [Shapefile](http://switchfromshapefile.org) in <MainPlatformName />, as we do not support collaborative use for this format.
```
Comment (nyalldawson) [inline src/migrate/arcgis/index.md:70]: ArcGIS Pro is **really** finicky about geopackage. It works ok sometimes, other times for no discernable reason it will refuse to open a gpkg. 

I would reword to this say "ArcGIS Pro has limited support for Geopackage"

File: src/migrate/arcgis/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+There are multiple options for data conversion. We recommend using [SLYR](#slyr) processing algorithms to convert project data to GeoPackage (GPKG). You can also use QGIS *Convert format* or *Package layers* processing algorithms. Alternatively, export your data to [formats supported by <MainPlatformName />](../../gis/supported_formats.md) in ArcGIS Pro, if you feel more comfortable doing it there. Data conversion can be done also in the console using the `ogr2ogr` command line tool.
+
+## Hybrid setup: using Mergin Maps with ArcGIS
+
+It is possible to use ArcGIS and <MainPlatformName /> in a hybrid setup. If you store your data in PostgreSQL/PostGIS database, you can use our [DB Sync](../../dev/dbsync/) tool to synchronise project data from PostgreSQL to <MainPlatformName /> (i.e. storing data in GeoPackages on <MainPlatformName /> server). You still need to have a QGIS project to style your GeoPackage data and set up the project (see the [Migrate section](#migrate-from-arcgis-to-qgis)).
```
Comment (nyalldawson) [inline src/migrate/arcgis/index.md:76]: The same should work for Geodatabases 👍

File: src/migrate/arcgis/index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+| ArcGIS Maps SDK | [Python client](../../dev/integration/#python-client-module) and QGIS API| developer SDK |
+
+*ArcGIS Pro replaced [ArcMap](https://en.wikipedia.org/wiki/ArcMap)
+**ArcGIS Field Maps replaced ArcGIS Collector since the end of [2021](https://www.Esri.com/arcgis-blog/products/collector/field-mobility/arcgis-collector-on-windows-platform-is-retired/)
+
+## Migrate from ArcGIS to QGIS
+
+To migrate the project, we recommend to: 
+1. Use the [**SLYR**](#slyr) plugin to transfer the ArcGIS project to QGIS project (including styling, etc.).
+2. Convert your survey (vector) layers (e.g from Shapefile or File Geodatabase) to GeoPackage if not done already by SLYR
```
[the remark this answers is not in the document]
Comment (PeterPetrik) [reply src/migrate/arcgis/index.md:37]: The problem is that ONLY geopackage (or postgis) can be used in Mergin Maps for collaborative work. If you use any other format for vector data the work from multiple surveyors will not be merged together but they will overwrite each other changes in worst case.

File: src/migrate/arcgis/index.md (hunk 3 above)
Comment (PeterPetrik) [reply src/migrate/arcgis/index.md:66]: When you say out of the box, you mean some processing algrorithm in QGIS?

File: src/migrate/arcgis/index.md (hunk 5 above)
Comment (PeterPetrik) [reply src/migrate/arcgis/index.md:83]: It will not work again because Mergin Maps (geodiff) does not support geodatabases for collaborative work.

**Comment by PeterPetrik** [comment]
@nyalldawson thanks for the review :) ... hopefully more people will discover SLYR this way

File: src/migrate/arcgis/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (nyalldawson) [reply src/migrate/arcgis/index.md]: Ok, that makes sense then!

File: src/migrate/arcgis/index.md (hunk 3 above)
Comment (nyalldawson) [reply src/migrate/arcgis/index.md]: Nope, just by loading the layer into QGIS any field domains detected will get immediately converted to value map widgets. This works with any format which supports the GDAL field domain API, which includes both Geodatabases and Geopackage.
